components/dashboard/src/settings/Preferences.tsx

```diff
@@ -19,6 +18,15 @@ import settingsMenu from "./settings-menu";
 
 type Theme = 'light' | 'dark' | 'system';
 
+const DesktopNoneId = "none"
+const DesktopNone: IDEOption = {
+    "image": "",
+    "logo": "",
```

Comment:
**suggestion:** Pasting below a possible logo option we could use here.

```svg
<svg width="64" height="64" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path fill-rule="evenodd" clip-rule="evenodd" d="M50.102 13.898c-9.997-9.997-26.206-9.997-36.204 0-9.997 9.997-9.997 26.207 0 36.204s26.207 9.997 36.204 0 9.997-26.206 0-36.204ZM36.526 41.051a3.2 3.2 0 0 0 4.525-4.526L27.474 22.95a3.2 3.2 0 0 0-4.525 4.525l13.576 13.577Z" fill="#D6D3D1"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h64v64H0z"/></clipPath></defs></svg>
```

| None option logo |
|-|
| <img width="700" alt="IDE-options" src="https://user-images.githubusercontent.com/120486/149943772-809fd3f8-ae0c-4c53-9a40-39fa931196ca.png"> |

Reply:
@mustard-mh I think so, but not entirely sure. See https://github.com/gitpod-io/gitpod/pull/7141 and https://github.com/gitpod-io/gitpod/issues/6955. Cc @iQQBot @akosyakov

Reply:
It is fine here as well. I don't think we are going to change it often. We put logos on ide proxy to add new IDEs without involving meta team. General changes of preferences UI are deployed by meta.

Reply:
Already changed, please review again

Reply:
![image](https://user-images.githubusercontent.com/20944364/149949820-7860a1e7-d3d3-40dd-8859-76d1ea2021a0.png)
![image](https://user-images.githubusercontent.com/20944364/149949932-aaa65a25-e416-40f0-a420-8c4dc35e96ca.png)
SVGs looks incongruous (none got a padding?)

Reply:
Thanks for noticing, @mustard-mh! ✨ 

The :a: extra padding was added to make this icon less prominent and not stand out as much as the rest of the other editor icons as well as :b: provide some visual balance next to other editor icons (logos) which don't use the whole icon bounding box. If you notice the overall size of all of them still remains the same (64x64px) in order to maintain consistent layout.

| Using a different live area (PROPOSED) | Using bounding box size (ALTERNATIVE) |
|-|-|
| <img width="700" alt="Preferences" src="https://user-images.githubusercontent.com/120486/149952465-937a856d-4c63-4f01-9b2d-ec34ffa06c29.png"> | <img width="700" alt="Preferences-1" src="https://user-images.githubusercontent.com/120486/149952469-60ca9a17-363b-41fd-ac42-cb0b41035dfb.png"> |

Reply:
It looks more stand out in dark mode
![image](https://user-images.githubusercontent.com/20944364/149952963-7c7663ce-967b-49f4-a9bb-edbf5e2bc3e3.png)

Reply:
Nice catch, @mustard-mh! ⚾ 

I'd be certain the grayscale filter could help but not in this case. 💭 

We could use different versions with different colors but a boring (simple) solution here could be to just use a different color for both. Let's go with that since we're probably going to update this section anyway.

```svg
<svg width="64" height="64" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path fill-rule="evenodd" clip-rule="evenodd" d="M50.102 13.898c-9.998-9.997-26.207-9.997-36.204 0s-9.997 26.207 0 36.204 26.206 9.997 36.204 0c9.997-9.997 9.997-26.206 0-36.204ZM36.525 41.051a3.2 3.2 0 0 0 4.526-4.526L27.474 22.95a3.2 3.2 0 1 0-4.525 4.525l13.576 13.577Z" fill="#A8A29E"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h64v64H0z"/></clipPath></defs></svg>
```

| Light | Dark |
|-|-|
| <img width="737" alt="desktop-light" src="https://user-images.githubusercontent.com/120486/149953994-ecedca5f-9b11-4ea2-a51c-b55707733602.png"> | <img width="737" alt="desktop-dark" src="https://user-images.githubusercontent.com/120486/149954011-5f636e40-5df4-4785-92a0-3dec3d6305cd.png"> |

Reply:
let it go, since it is not a big problem

Reply:
hey @gtsiolis 👋, just make dark logo visible and remove some code